tools/rail_inspector/README.md

```diff
@@ -0,0 +1,46 @@
+# RailInspector
+
+A collection of linters for [`rails/rails`](https://github.com/rails/rails)
+
+## Installation
+
+Install the gem and add to the application's Gemfile by executing:
+
+    $ bundle add rail_inspector
+
+If bundler is not being used to manage dependencies, install the gem by
+executing:
+
+    $ gem install rail_inspector
```

Comment:
Do we want to still release it as a gem? 🤔

Reply:
Removed README because I don't think it was adding much